graphx/src/main/scala/org/apache/spark/graphx/lib/StronglyConnectedComponents.scala

```diff
@@ -106,6 +116,16 @@ object StronglyConnectedComponents {
           }
         },
         (final1, final2) => final1 || final2)
+
+        // ensure sccGraph's rdd are marked as recently used
+        sccGraph.vertices.count()
+        sccGraph.edges.count()
+        // sccGraph materialized so, unpersist can be done on previous
+        if(prevSccGraph != sccGraph)
```

Comment:
I think this will fail style checks but let's see.
Hm, I think it's straightforward to follow the same pattern for the other RDD, but yes it's updated in enough places that it's a fair bit of extra code. It might be possible to refactor this nicely in some method.
Not essential, maybe. This code isn't used much anyway. A partial improvement may be a sufficient win.